### PR TITLE
Add debug to all Spack cmds

### PR DESCRIPTION
**Pull request by chapman39**
Debug each spack command if specified rather than spec and install

### CODE REVIEW

**Review by cyrush — APPROVED** [review]

File: uberenv.py [hunk 1]
```diff
@@ -770,11 +776,7 @@ def find_spack_pkg_path(self, pkg_name, spec = ""):
 
     # Extract the first line of the full spec
     def read_spack_full_spec(self,pkg_name,spec):
-        debug = ""
-        if self.opts["spack_debug"]:
-            debug = "--debug --stacktrace "
-
-        res, out = sexe("{0} {1}spec {2}".format(self.spack_exe(),debug,self.pkg_name_with_spec), ret_output=True)
+        res, out = sexe("{0} spec {2}".format(self.spack_exe(), self.pkg_name_with_spec), ret_output=True)
```
Comment (white238) [inline uberenv.py:779]:
```suggestion
        res, out = sexe("{0} spec {1}".format(self.spack_exe(), self.pkg_name_with_spec), ret_output=True)
```

Comment (chapman39) [reply uberenv.py:779]: Thanks for catching this. This function is never being called anywhere by the way-- not sure if we need to keep it or not.

Comment (white238) [reply uberenv.py]: If it isn't being called you can just remove it.

**Comment by chapman39** [comment]
Fixes #87

**Pull request merged**